README.md

```diff
@@ -1,21 +1,32 @@
 # Honey-Dos
 
+## Prerequisites
```

Comment:
@jantaylor I'd like your input on these prerequisites, is linux the same?

Reply:
Those are the same for Linux.